### ISSUE TITLE
CheckA11y - html reporter prevents test from failing

### ISSUE THREAD

**Issue by egor-limenko** [abhinaba-ghosh/axe-playwright#223]
I have the following usage of `checkA11y` function
```
await checkA11y(
      page,
      undefined,
      {
        axeOptions: {
          runOnly: {
            type: 'tag',
            values: ['wcag2a', 'wcag2aa', 'wcag21a', 'wcag21aa', 'wcag22aa'],
          },
        },
      },
      false,
      'html',
      {
        outputDirPath: 'results',
        outputDir: 'accessibility-report',
        reportFileName: 'results.html',
      },
    )
 ```
 As you can see, I specified `false` for skipFailures argument. But as soon as I specify "html" reported tests no longer fail. How can I have both "html" as a reporter and tests fail in case of a11y errors?
Thanks in advance
